src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.vsmanproj

```diff
@@ -6,6 +6,7 @@
     <FinalizeManifest>true</FinalizeManifest>
     <FinalizeSkipLayout>true</FinalizeSkipLayout>
     <OutputPath>$(OutDir)Insertion\</OutputPath>
+    <ManifestPublishUrl>https://vsdrop.corp.microsoft.com/file/v1/Products/DevDiv/dotnet/roslyn/master/20160729.6;</ManifestPublishUrl>
```

Comment:
Actually this is necessary.  We need a ManifestPublishUrl in order to pass validation.